src/Resources/app/administration/src/module/findologic-module/page/findologic-page/index.js

```diff
@@ -145,6 +150,30 @@ Component.register('findologic-page', {
             }
         },
 
+        /**
+         * Updates the header bar link, based on the used Shopware version.
+         *
+         * * Shopware >= 6.4 => Extension list.
+         * * Shopware < 6.4 => Settings page.
+         */
+        updateHeaderBarBackLink() {
+            let foundModule = null;
+            Shopware.Module.getModuleRegistry().forEach((module) => {
+                if (foundModule) {
+                    return;
+                }
+
+                if (module.manifest.title === 'sw-extension-store.title') {
+                    foundModule = module;
```

Comment:
Return here when foundModule is set?

Reply:
You can't return inside a `.forEach` in JavaScript. That's also how Shopware solves this in their own code.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach#description

> There is no way to stop or break a forEach() loop other than by throwing an exception.